content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,109 @@
+===============================================================
+Meetup 2023 September 20th in NYCU (do-unknown-things training)
+===============================================================
+
+:date: 2023-08-31 13:00
+:url: meetup/2023/0920-nycu
+:save_as: meetup/2023/0920-nycu.html
+
+
+Date
+-----
+
+* Date: September 20th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:00 Checking in
+* 19:00-20:30 Training
+* 20:30-21:30 Meeting/ Free discussion
+
+|
+
+Subjects
+------------------
+
+Training: do unknown things
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Speaker: 
+Yung-Yu - A numerical analyst and HPC software architect.  He has experience in
+different application domains, including computational fluid dynamics (CFD) and
+computational lithography.  Yung-Yu is currently `working
+<https://www.linkedin.com/in/yungyuc/>`__ as a technologist of OPC at Synopsys,
+and spending some time `teaching
+<https://yyc.solvcon.net/en/latest/nsd/index.html>`__ software development for
+engineering at NYCU.  He has contributed to the Python community for many years
+and is a PSF fellow.
+
+In this training, Yung-Yu will talk about practical practices that are applied to general 
```

Comment:
Please reorder the content in the training section: (i) introduction to the training (this passage), (ii) time table, and (iii) introduction to the trainer.

Reply:
@yungyuc, I wonder whether the "trainer" refers to you, and "introduction to the trainer" should be exactly what is currently under "speaker"?

Reply:
Yes, trainer refers to me.  The person who conducts a training is a trainer, not a speaker.

Reply:
Okay, let me remove the speaker section and add trainer introduction to the training section.

Reply:
My request was to reorder the content without removing anything.

Reply:
@yungyuc, I have reordered the content. Can you please take a look at it?